cli/Cargo.toml

```diff
@@ -22,7 +22,7 @@ flate2 = { version = "1.0.22" }
 zip = { version = "0.5.13", default-features = false, features = ["time", "deflate"] }
 regex = { version = "1.5.5" }
 lazy_static = { version = "1.4.0" }
-sysinfo = { version = "0.23.5" }
+sysinfo = { version = "0.27.7" }
```

Comment:
Just a small update since there was a warning that 0.23 would no longer work in newer Rust versions.